README.md

```diff
@@ -2,55 +2,46 @@
 
 ## Demo
 
-> This showcase swaps on Uniswap with a Passkey-verified account contract. For more demos, visit the [demo](demo.md) page.
+> This showcase swaps on Uniswap with a Passkey-verified account contract.
 
 [![Waallet Demo](https://i.imgur.com/ojApmiQ.gif)](https://youtu.be/9D5K4IlofLY)
 
-## Development
+## Testing
 
-This project depends on [waallet-contract](https://github.com/pilagod/waallet-contract) to provide the testnet environment, please follow the instructions in the repository to setup local testnet.
+This project depends on [waallet-contract](https://github.com/pilagod/waallet-contract) to provide the network environment, please follow the instructions in the repository to setup local testnet.
 
-After you spin up a testnet from [waallet-contract](https://github.com/pilagod/waallet-contract), run following commands to do testing:
+After you spin up a testnet from [waallet-contract](https://github.com/pilagod/waallet-contract), run the following command to do testing:
 
 ```bash
 npm run test
 ```
 
-## Plasmo Extension
+## Extension
 
-This is a [Plasmo extension](https://docs.plasmo.com/) project bootstrapped with [`plasmo init`](https://www.npmjs.com/package/plasmo).
+> [!NOTE]  
+> This project is bootstrapped with [Plasmo](https://docs.plasmo.com/).
 
-### Getting Started
+### Preapre Environment Variables
 
-First, run the development server:
+Please refer to `.env.development.example` and preapre your own `.env.development`.
```

Comment:
typo 
```suggestion
Please refer to `.env.development.example` and prepare your own `.env.development`.
```

Reply:
Fixed in [7dcb60f](https://github.com/pilagod/waallet-extension/pull/196/commits/7dcb60fceaf8813df3710b840b8db858da7a1a9f) 🙈